.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+# Summary of the issue
+...
+
+## Expected behavior
+...
+
+## Steps to reproduce
+...
+
+*Where applicable, consider providing a patch that uses the [end to end testing environment](https://github.com/seisollc/zeek-kafka/blob/main/docker/run_end_to_end.sh).*
+
+## Logs, errors, etc.
+...
+
+## Your environment
+* Version of Zeek
+* Version or commit hash of the zeek-kafka package
+* Operating System and version
```

Comment:
* docker version 

maybe we want system info script?

Reply:
That makes sense but I would rather handle as a follow-on.  Can you open an issue?